tcpdf.php

```diff
@@ -16396,8 +16396,8 @@ protected function getHtmlDomArray($html) {
 		// create a special tag to contain the CSS array (used for table content)
 		$csstagarray = '<cssarray>'.htmlentities(json_encode($css)).'</cssarray>';
 		// remove head and style blocks
-		$html = preg_replace('/<head([^\>]*)>(.*)?<\/head>/siU', '', $html);
-		$html = preg_replace('/<style([^\>]*)>([^\<]*)<\/style>/isU', '', $html);
+		$html = preg_replace('/<head([^\>]*?)>(.*)<\/head>/is', '', $html);
```

Comment:
I belive this will bring back the #267 issue

@eisberg please test

Reply:
Yes, I tested, with this option, an empty file is generated. This bring back the #267 issue

```
$html = preg_replace('/<head([^\>]*)>(.*)?<\/head>/siU', '', $html); // Works correctly
```

```
$html = preg_replace('/<style([^\>]*?)>([^\<]*?)<\/style>/is', '', $html); // It doesn't work properly, the content is deleted
```

Reply:
resolved in https://github.com/tecnickcom/TCPDF/pull/359#discussion_r608555448